examples/2d/sprite_sheet.rs

```diff
@@ -41,13 +49,16 @@ fn setup(
     let texture_atlas =
         TextureAtlas::from_grid(texture_handle, Vec2::new(24.0, 24.0), 7, 1, None, None);
     let texture_atlas_handle = texture_atlases.add(texture_atlas);
+    let animation_indices = AnimationIndices { first: 1, last: 6 };
```

Comment:
```suggestion
    // Use only the subset of sprites in the sheet that make up the run animation
    let animation_indices = AnimationIndices { first: 1, last: 6 };
```
I think there's some potential first confusion about how the sprites are indexed here. A comment explaining why we're doing this might help.